components/common/listingCart/ListingCartModal.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
+    <NeoModal
+      v-model="preferencesStore.listingCartModalOpen"
+      :can-cancel="['outside', 'escape']"
```

Comment:
why do you need to specify?

default value for `can-cancel` is true

Reply:
<img width="417" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/4a1655ff-471c-41b5-875f-827ce51d0a2c">
It's the same with the other components using it.
I just want to be consistent

Reply:
removing `can-cancel` prop causes no change in beavior
please remove it